che-incubator/chectl
# feat: use name as username claim and add more users to dex on minikube

Signed-off-by: Michal Vala <mvala@redhat.com>



### What does this PR do?
 - use `name` as username claim when using dex with minikube. This ends up as user name in Kubernetes to bind roles.
 - add more users for testing (user1-5 with `password`  password)

### Screenshot/screencast of this PR



### What issues does this PR fix or reference?

part of https://github.com/eclipse/che/issues/20635

### How to test this PR?

It's not easy to test it. You need to deploy some app that will be configured and utilize OIDC settings. For example app from here https://github.com/che-incubator/che-auth-playground/tree/master/minikube_dex. You should be able to login with added users and control RBAC with username instead of an email.


### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.
